_layouts/report.html

```diff
@@ -16,6 +20,11 @@ <h1 class="title page-title">Report: {{page.title}}</h1>
     <hr class="divider" />
     <p>
         Thank you for reading this report, <b>{{page.title}}</b>. It was authored by the Teach LA community, with significant contributions from {{page.author}}.
+        </br></br>Subscribe to our RSS feed - 
```

Comment:
I would avoid using `</br>` for spacing; let's instead use `margin` or `padding` if possible!